src/components/products-section/ProductsSection.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import Product from "../product/Product";
+import "./style.css";
+
+const ProductsSection = ({ title, products, category }) => {
+  return (
+    <>
+      <section className={`products-section products-section--${category}`}>
```

Comment:
Tohle je už takový pro-tip. Pokud to jen trochu jde (=mimo velice speciální případy), tak se názvy tříd pokus držet vcelku. To všechno kvůli jedné potřebě - když budeš chtít zjistit kde v aplikaci používáš ".products-section--dresses". Pokud ty názvy budeš mít vždy vcelku, tak si jen prostě dáš "najít string ".products-section--dresses"" a hotovo. Pokud to takhle kouskuješ, tak se ti to začne hledat těžko => budeš řešit více bugů.

Takže v tomhle případě bych udělal:
```
const categoryClasses = {
  dresses: `products-section--dresses`,
  ...
}
...
<section className={`products-section ${categoryClasses[category]}`}>
```